e2e/test/scenarios/models/reproductions.cy.spec.ts

```diff
@@ -816,11 +816,10 @@ describe("issue 34574", () => {
     cy.findByTestId("sidebar-right").within(() => {
       cy.log("Set the model description to a markdown text");
       cy.intercept("PUT", `/api/card/${ORDERS_MODEL_ID}`).as("updateModel");
-      cy.findByPlaceholderText("Add description")
-        .type(
-          "# Hello{enter}## World{enter}This is an **important** description!",
-        )
-        .blur();
+      cy.findByPlaceholderText("Add description").type(
+        "# Hello{enter}## World{enter}This is an **important** description!",
+      );
+      cy.realPress("Tab");
```

Comment:
Using `.blur()` failed about 20% of the times in the stress-test scenario.
https://github.com/metabase/metabase/actions/runs/9891293400/job/27321408753#step:9:261